### ISSUE TITLE
Network attributes error logs when using Talos OS

### ISSUE THREAD

**Issue by aivanov-citc** [kubernetes-sigs/node-feature-discovery#1842]
We use Talos as the operating system for kubernetes. NFD logs constantly show errors related to the network attributes of bonding_masters.

**What happened**:
```
E0819 06:20:51.415724       1 network.go:154] "failed to read net iface attribute" err="open /host-sys/class/net/bonding_masters/operstate: not a directory" attributeName="operstate"
E0819 06:20:51.415775       1 network.go:154] "failed to read net iface attribute" err="open /host-sys/class/net/bonding_masters/speed: not a directory" attributeName="speed"
I0819 06:20:51.816548       1 nfd-worker.go:664] "starting feature discovery..."
I0819 06:20:51.816977       1 nfd-worker.go:677] "feature discovery completed"
```

**What you expected to happen**:
Can you fix it so that this interface is ignored during scanning

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): v1.30.3
- OS (e.g: `cat /etc/os-release`): Talos 1.7.6
- Kernel (e.g. `uname -a`): 6.6.43-talos

**Comment by marquiz** [comment]
@aivanov-citc thanks for reporting the issue. Yes, I think we should ignore the `bonding_masters` "iface". The error itself is harmless even if annoying and distractive.
/help

**Comment by jfroy** [comment]
This does not appear to be fixed in 0.16.6, yet the PR/commit/patch is in that release.

```
node-feature-discovery-worker-xddcc worker I1114 19:20:50.091990       1 nfd-worker.go:293] "Node Feature Discovery Worker" version="v0.16.6" nodeName="kantai1" namespace="kube-system"
node-feature-discovery-worker-xddcc worker I1114 19:20:50.092188       1 nfd-worker.go:622] "configuration file parsed" path="/etc/kubernetes/node-feature-discovery/nfd-worker.conf"
node-feature-discovery-worker-xddcc worker I1114 19:20:50.092458       1 nfd-worker.go:654] "configuration successfully updated" configuration={"Core":{"Klog":{},"LabelWhiteList":"","NoPublish":false,"FeatureSources":["all"],"Sources":null,"LabelSources":["all"],"SleepInterval":{"Duration":60000000000}},"Sources":{"cpu":{"cpuid":{"attributeBlacklist":["AVX10","BMI1","BMI2","CLMUL","CMOV","CX16","ERMS","F16C","HTT","LZCNT","MMX","MMXEXT","NX","POPCNT","RDRAND","RDSEED","RDTSCP","SGX","SGXLC","SSE","SSE2","SSE3","SSE4","SSE42","SSSE3","TDX_GUEST"]}},"custom":[],"fake":{"labels":{"fakefeature1":"true","fakefeature2":"true","fakefeature3":"true"},"flagFeatures":["flag_1","flag_2","flag_3"],"attributeFeatures":{"attr_1":"true","attr_2":"false","attr_3":"10"},"instanceFeatures":[{"attr_1":"true","attr_2":"false","attr_3":"10","attr_4":"foobar","name":"instance_1"},{"attr_1":"true","attr_2":"true","attr_3":"100","name":"instance_2"},{"name":"instance_3"}]},"kernel":{"KconfigFile":"","configOpts":["NO_HZ","NO_HZ_IDLE","NO_HZ_FULL","PREEMPT"]},"local":{},"pci":{"deviceClassWhitelist":["03","0b40","12"],"deviceLabelFields":["class","vendor"]},"usb":{"deviceClassWhitelist":["0e","ef","fe","ff"],"deviceLabelFields":["class","vendor","device"]}}}
node-feature-discovery-worker-xddcc worker I1114 19:20:50.102088       1 metrics.go:44] "metrics server starting" port=":8081"
node-feature-discovery-worker-xddcc worker E1114 19:20:50.110661       1 network.go:154] "failed to read net iface attribute" err="open /host-sys/class/net/bonding_masters/operstate: not a directory" attributeName="operstate"
node-feature-discovery-worker-xddcc worker E1114 19:20:50.110689       1 network.go:154] "failed to read net iface attribute" err="open /host-sys/class/net/bonding_masters/speed: not a directory" attributeName="speed"
```

**Comment by aivanov-citc** [comment]
v0.17.1, it still displays an error
```logs
E0129 05:38:34.863250       1 network.go:154] "failed to read net iface attribute" err="open /host-sys/class/net/bonding_masters/operstate: not a directory" attributeName="operstate"
E0129 05:38:34.863295       1 network.go:154] "failed to read net iface attribute" err="open /host-sys/class/net/bonding_masters/speed: not a directory" attributeName="speed"
I0129 05:38:34.918859       1 nfd-worker.go:664] "starting feature discovery..."
I0129 05:38:34.919028       1 nfd-worker.go:677] "feature discovery completed"
```
